posthog/api/__init__.py

```diff
@@ -128,6 +128,7 @@ def api_not_found(request):
     "historical_exports",
     ["team_id", "plugin_config_id"],
 )
+webhooks_metrics_router = projects_router.register("webhooks", app_metrics.WebhooksViewSet, "webhooks", ["team_id"])
```

Comment:
I'm not sure how closely related to apps this endpoint is in practice, but it should definitely not be under `/projects/:id/webhooks/`, since that would be intended for actual Webhook resources – webhook _metrics_ should have "metrics" somewhere in the URL.

I also suggest using "hooks" to cover both webhooks and REST hooks, since people are often surprised that these are two separate mechanisms. But they are both webhooks in principle so… 🤷 

```suggestion
app_metrics_router.register("hooks", app_metrics.WebhooksViewSet, "hooks", ["team_id"])
```

Reply:
so `hooks_metrics` ?

Reply:
Yeah, maybe `/projects/:id/hook_metrics/`

Reply:
I suggested nesting this under `app_metrics` here, but I see now that this doesn't wouldn't work very well with the way we're (ab)using the app metrics mechanism